Don't run all the portindex processes simultaneously. There are so many of them
now that the system can't keep up with other processes, like the web server. A
solution like running just a few portindex processes in parallel using GNU
parallel could be investigated later.
--- mprsyncup.orig	2024-03-12 21:30:37.000000000 -0500
+++ mprsyncup	2024-03-12 21:30:37.000000000 -0500
@@ -201,9 +201,8 @@
             INDEX="PortIndex_darwin_${PLATFORM}"
             ${PORTINDEX} -p "macosx_${PLATFORM}" -o "${INDEX}" \
                 | ${AWK} '{ print "Updating " idx ":\t" $0 }' idx="$INDEX" \
-                | expand -t 40,48,56,64,72,80 &
+                | expand -t 40,48,56,64,72,80
         done
-        wait
 
         # generate json for each platform-specific index
         for PLATFORM in $PLATFORMS; do
